boards/beaglev/fire/demos-and-tutorials/gateware/customize-cape-gateware-verilog.rst

```diff
@@ -82,13 +82,12 @@ call our custom cape gateware ``MY_LOVELY_CAPE``.
 
         HSS:
             type: git
-            link: https://git.beagleboard.org/beaglev-fire/hart-software-services.git
-            branch: develop-beaglev-fire
-            board: bvf
+            link: https://github.com/polarfire-soc/hart-software-services.git
+            branch: master
```

Comment:
In general, if we could avoid the branch name 'master', it would be best.

Reply:
Third party project